config/twill-navigation.php

```diff
@@ -202,9 +202,13 @@
     ],
     'generic' => [
         'title' => 'Pages',
-        'route' => 'admin.generic.genericPages.index',
+        'route' => 'admin.generic.landingPages.index',
```

Comment:
We will have the concept of a a generic page alongside landing pages. So rather than a replace these two lines should live together. ("Generic Pages" are different from "Pages").